src/main/webapp/script.js

```diff
@@ -1,15 +1,26 @@
 async function displayCodes() {
     var formInput = document.getElementById('formInput').value;
-    console.log("HELLO");
-    const response = await fetch('/promo-code?formInput=' + formInput);
+    const selected = document.querySelector('input[name="searchOption"]:checked');
+    var response; 
```

Comment:
The `var` keyword is discouraged and using `let` or `const` is recommended by the [style guide](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let).